src/style.css

```diff
@@ -1 +1,62 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  padding: 0;
+  margin: 20px 0;
```

Comment:
You have both top and bottom margins set here. According to the checklist, you should be consistent with your margins and add only top or bottom, but not both, to avoid potential margin collapse.